src/monitoring_service/states.py

```diff
@@ -106,7 +107,8 @@ def __init__(
 
         if signature is None:
             assert priv_key
-            self.signature = encode_hex(eth_sign(priv_key, self.serialize_bin()))
+            signer = LocalSigner(private_key=to_bytes(hexstr=priv_key))
```

Comment:
What is the difference between `to_bytes(hexstr=priv_key)` and `decode_hex(priv_key)`?

Reply:
`decode_hex`  is nicer ;)